dataactcore/interfaces/function_bag.py

```diff
@@ -334,13 +336,19 @@ def check_job_dependencies(job_id):
                 # so it is eligible to be set to a 'ready'
                 # status and added to the queue
                 mark_job_status(dep_job_id, 'ready')
-                # add dep_job_id to the SQS job queue
 
-                logger.info('Sending job %s to job manager in sqs', dep_job_id)
-                sqs = boto3.resource('sqs', region_name=CONFIG_BROKER['aws_region'])
-                queue = sqs.get_queue_by_name(QueueName='job-manager')
-                response = queue.send_message(MessageBody=str(dep_job_id))
-                logger.info('Send message response: %s', response)
+                # If local, utilize local db to mock queue
+                if g.is_local:
```

Comment:
Why are we not logging anything if it's local? I would think we still need some indication just in case something goes wrong.